src/index.js

```diff
@@ -0,0 +1,312 @@
+"use strict"
+// GLOBAL VARIABLES
+const TEMP_DATA = {
+  hot: {
+    F: 80,
+    C: 27,
+    icon: '🐍  🦂 🌵 🌵 🐍  🏜  🦂 🌵'
+  },
```

Comment:
Looks like you use the temp description string ("hot") and the value from `F` to assign a class color in `setColorAndLandscape`. 

For example:
```js
if (state.tempValue >= TEMP_DATA.hot.F) {
     state.temp.className = '${temp.className} red';
     state.landscape.textContent = TEMP_DATA.hot.icon;
}
```

Instead of adding `red` in the `setColorAndLandscape`, could you store that color value in the object here too and then get the color value with dot notation from the object?

```suggestion
  hot: {
    F: 80,
    C: 27,
    icon: '🐍  🦂 🌵 🌵 🐍  🏜  🦂 🌵',
    color: 'red'
  },
```

So that your `setColorAndLandscape` logic would be like 
```js
if (state.tempValue >= TEMP_DATA.hot.F) {
     state.temp.className = '${temp.className} {TEMP_DATA.hot.color}; // get color from object
     state.landscape.textContent = TEMP_DATA.hot.icon;
}
```

Just thinking it's more maintainable to change the colors in your object than it is to comb through a helper function and carefully change the colors where they need changing.